roles/brotli_nginx/tasks/build.yml

```diff
@@ -47,7 +47,7 @@
   changed_when: False
 
 - name: configure build # noqa 305
-  shell: "./configure {{ nginx_build_args }} --with-compat --add-dynamic-module={{ nginx_brotli_src_path }}"
+  shell: "./configure {{ nginx_build_args }} --add-dynamic-module={{ nginx_brotli_src_path }}"
```

Comment:
We don't have any issues with this on Ubuntu 16 or 18... where you using Ubuntu 20 while testing? Was it a standard DO Droplet?

Reply:
I don't remember if we agreed to go straight to Ubuntu 20 but it could as well be the standard.